admin_manual/configuration/server/occ_command.rst

```diff
@@ -950,7 +950,62 @@ See :doc:`../../issues/code_signing` to learn more.
 l10n, Create Javascript Translation Files for Apps
 --------------------------------------------------
 
-This command is for app developers to update their translation mechanism from ownCloud 7 to ownCloud 8 and later.
```

Comment:
I guess we can simply from this command - it was used when migrating from owncloud 7 to 8 for apps which don't deliver translation files in the required format.

this is of zero use as of today. I'll create a PR to kill the command in core

Reply:
Thanks for the update @DeepDiver1975